### DISCUSSION TITLE
Does the library work with displays connected via RGB parallel interface

### DISCUSSION THREAD

**Discussion by brightproject** [Bodmer/TFT_eSPI#2760] [General]
I have a good display with `esp32-s3` on one board `SmartPanle_ZX4D30NE01S`.
![board1](https://github.com/Bodmer/TFT_eSPI/assets/1788098/d0eecaa1-b46c-475a-96a5-ecb6cce3d899)
It looks like it has a MIPI interface.
I wanted to use this hardware for my projects, but because I use the TFT_eSPI library, so I can’t connect the display yet.
![WhatsApp Image 2023-07-04 at 11 18 23](https://github.com/Bodmer/TFT_eSPI/assets/1788098/9a7ed270-22df-4abe-923d-be2a027ae57c)
Scheme available
[WT32S3-43L（ZX4D30NE01S-UR-4827） Datasheet-V1.2EN(1).pdf](https://github.com/Bodmer/TFT_eSPI/files/12162735/WT32S3-43L.ZX4D30NE01S-UR-4827.Datasheet-V1.2EN.1.pdf)
Also an interesting [article](https://riverdi.com/blog/the-parallel-rgb-interface-explained/) I read.

**Comment by Bodmer** [comment]
The RGB parallel interface is not supported. The problem is the high data rate required (circa 8Mpixels/s), this requires hardware support to be built into a high performance processor with sufficient RAM for whole frame buffer.

There are no plans to add 24 bit RGB parallel support to TFT_eSPI.

**Comment by brightproject** [comment]
I met that the `RGB` interface is also called `MIPI.`
Initially, in this module, based on `esp32-c3,` the firmware was loaded from [here.](https://github.com/wireless-tag-com/ZX4D30NE01S/tree/master)
This [sketch](https://github.com/wireless-tag-com/ZX4D30NE01S/blob/master/components/lvgl/examples/arduino/LVGL_Arduino/LVGL_Arduino.ino) even features your library, because `lvgl` , as I understand it, is a framework that uses your core for rendering.
I'm trying to figure out how to adapt your [library](https://github.com/Bodmer/TFT_eSPI) to the RGB interface.

> https://github.com/wireless-tag-com/ZX4D30NE01S/blob/5f481818c4e88bf9d53d1f99ad09ecf3d69fd803/main/screen.c

> https://github.com/wireless-tag-com/ZX4D30NE01S/blob/5f481818c4e88bf9d53d1f99ad09ecf3d69fd803/main/board.h
